Makefile

```diff
@@ -19,13 +19,15 @@ OST_VERSION := $(shell go list -m github.com/Finschia/ostracon | sed 's:.* ::')
 DOCKER := $(shell which docker)
 BUILDDIR ?= $(CURDIR)/build
 CGO_ENABLED ?= 1
-ARCH ?= x86_64
+ARCH ?= amd64
 TARGET_PLATFORM ?= linux/amd64
+TARGET_ARCH ?= x86_64
```

Comment:
`TARGET_ARCH` should not accept the user input, because a user may set `ARCH=amd64` and `TARGET_ARCH=amd64`. There is no logic updating `TARGET_ARCH` according to the corresponding `ARCH`.
So, it would be better to change the line into `TARGET_ARCH = x86_64` or insert the logic.

Reply:
It's a good point. 👍 
I changed `TARGET_ARCH` and `TARGET_PLATFORM` not being used as input